src/components/footer/footer.module.css

```diff
@@ -0,0 +1,39 @@
+.footerWrapper {
```

Comment:
Ensure the footer is always positioned at the bottom of the page.
<img width="637" alt="Screenshot 2024-04-27 at 23 08 07" src="https://github.com/alyona-shirpal/fe-react-2024/assets/102523692/a2dd6d90-1bc1-4668-852e-a27ad87e4d40">